### ISSUE TITLE
New Feature Request: When a group of photos are dragged, "cycle" only bettween them

### ISSUE THREAD

**Issue by jjarava** [galymzhan/skadi#9]
Hi!

Nice app!

One suggestion - if I've dragged-and-dropped a number of pictures into the app's window, when I do previous/next, I would expect the next/prev picture of the list I've dropped in the app, but not of the folder that contains the pictures.

So if I've dragged 10 pictures out of a folder with 1000, the full "prev/next" should cycle only between the 10?

If I've _just_ used 1 picture to get the app to focus on the folder, then it makes sense for the app to cycle between all images in the folder

**Comment by redcapital** [comment]
Yeah, this is exactly what I was intending to implement, but had no time. I'll try to add it
